docs/en/operations/table_engines/jdbc.md

```diff
@@ -20,11 +21,11 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name  ENGINE = JDBC(dbms_uri, external_d
     Example for MySQL: `jdbc:mysql://localhost:3306/?user=root&password=root`.
 
 - `external_database` — Database in an external DBMS.
-- `external_table` — A name of the table in `external_database`.
+- `external_table` — The name of the table in `external_database`.
 
 ## Usage Example
 
-Creating a table in MySQL (using native MySQL engine):
+Creating a table in MySQL (using the native MySQL engine):
```

Comment:
This description doesn't match the example, MySQL engine is a separate one and it has nothing to do with this JDBC one. Probably we should explicitly mention this and give a link. Or maybe change the example to use something that's not supported directly.

Reply:
Is not it obvious that we at firs step creating a table in MySQL and at the second step retreiving data in ClickHouse through JDBC?

Reply:
Now I see how this happened, so this should be something like "Creating a table in MySQL server by connecting directly with it's console client:"

It's important to get rid of confusing word "engine". Extra comment below when we switch back to ClickHouse might be useful as well.